packages/oc-template-react-compiler/lib/compileServer.js

```diff
@@ -51,12 +50,12 @@ module.exports = (options, callback) => {
       next =>
         fs.outputFile(higherOrderServerPath, higherOrderServerContent, next),
       next => compiler(config, next),
-      (data, next) => fs.remove(higherOrderServerPath, err => next(err, data)),
```

Comment:
if the compilation fails, the remove doesn't happen. So I moved this to the end of the chain.

Reply:
Nice!
I see 2 options for later:
a. don't remove this and mainly on compilation we scaffhold that file for the user (source-maps will therefore work, the same principle could be applied for the reactHOC
b. we transform this into a a webpack loader with the relative maps

Reply:
I would vote 2. - with 1, the user wouldn't set breakpoints on the compiled file - so I think 2 would be the more elegant solution